src/components/LoginForm/LoginForm.module.css

```diff
@@ -0,0 +1,138 @@
+
```

Comment:
Ogólnie sam formularz wygląda ok, jest walidacja - spoko. Grafiki też dobrze rozmieszczone na Desktop i tablet. 
Jednak całe zachowanie tego kontenera z zawartością, pozostawia wiele do życzenia: na większym ekranie strona jest jakby poucinana po bokach i z dołu (zrzut ekranu wysłany na Slack). Na tablecie ustawia się po lewej stronie i jest tam przyklejona (printscreen - Slack), nie sięga do dołu strony. Zrób RWD, aby tło rozlewało się na cały ekran, a elementy wewnątrz trzymały się środka. Na desktopie do max-width: 1600px. Wysokość powinna sięgać wszędzie do końca ekranu. Najlepiej wyśrodkować zawartość w pionie i poziomie. Zauważyłem dziwne zachowanie na szerokości 767px - wszystko znika. 
Przycisk LOGIN nie ma cursor: pointer. Po Kliknięciu w inputy, placeholdery ciągle sąwyświetlane. Jak zobaczysz. Sprawdź, jak ładnie zachowują się placeholdery w przypadku MUI - odbiór wizualny robi ogromne wrażenie na użytkowniku. 
Ponownie przejrzyj MUI - tam bardzo ładnie wyglądają przyciski, animacje - piękne są przejścia przy hover, focus i kliku.
Linki do zapoznania się:
https://codesandbox.io/s/formik-material-ui-and-yup-hy0ju 
https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwia0-H-nb-BAxWxgP0HHbUMB94QwqsBegQIEBAG&url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DaeU2nU45sw4&usg=AOvVaw03y63B5hb47ns9n2kz8AUk&opi=89978449